renovate.json

```diff
@@ -23,6 +23,15 @@
             "extends": [
                 "schedule:weekly"
             ]
+        },
+        {
+            "matchDatasources": [
+                "github-releases"
+            ],
+            "matchPackageNames": [
+                "OpenVPN/ovpn-dco-win"
+            ],
+            "allowedVersions": "2.x"
```

Comment:
Have you tested that this actually works? https://docs.renovatebot.com/configuration-options/#allowedversions sounds to me like it should be more like "<3.0"?

Reply:
Yes!

https://github.com/lstipakov/openvpn-build/pull/2